client/src/store/modules/scheduledEvent/actions.js

```diff
@@ -40,6 +40,7 @@ export default {
             eventEmails = [],
             eventStatus = [],
             tags = [],
+            searchString = null,
```

Comment:
string

Reply:
we should not transfer anything if we did not write anything to the request, because in API it will make unnecessary movements due to the fact that there is a line there (albeit empty)